helix-core/src/main/java/org/apache/helix/controller/stages/IntermediateStateCalcStage.java

```diff
@@ -421,7 +419,8 @@ private PartitionStateMap computeIntermediatePartitionState(ResourceControllerDa
     if (clusterStatusMonitor != null) {
       clusterStatusMonitor
           .updateRebalancerStats(resourceName, messagesForRecovery.size(), messagesForLoad.size(),
-              messagesThrottledForRecovery.size(), messagesThrottledForLoad.size());
+              messagesThrottledForRecovery.size(), messagesThrottledForLoad.size(),
+              onlyDownwardLoadBalance);
```

Comment:
Is  onlyDownwardLoadBalance same as ERROR partition count?

Reply:
TFTR. `onlyDownwardLoadBalance` is a 0-1 value metric. 1 means the load babalance is throttled because of current ERROR partition count > defined threshold.

Reply:
makes sense. thanks